Package.swift

```diff
@@ -18,6 +18,15 @@
  **/
 
 import PackageDescription
+import Foundation
+
+var kituraNetPackage: Package.Dependency
+
+if let _ = ProcessInfo.processInfo.environment["KITURA_NIO"] {
```

Comment:
I think I prefer `if ProcessInfo.processInfo.environment["KITURA_NIO"] != nil` like you did in the tests.